server/routers/pokemons/pokemons_api.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import APIRouter, HTTPException, status
+from Queries import queries
+import requests
+
+router = APIRouter()
+
+@router.get("/pokemons/{pokemon_name}/trainers")
+def get_trainers_of_pokemon(pokemon_name):
```

Comment:
async def get_trainers_of_pokemon(pokemon_name, status_code=200):
async cuse we talking to a database